external/elasticsearch/src/main/java/com/digitalpebble/stormcrawler/elasticsearch/metrics/StatusMetricsBolt.java

```diff
@@ -102,6 +103,7 @@ public Object getValueAndReset() {
     public void execute(Tuple input) {
         _collector.ack(input);
 
+        Metadata metadata = (Metadata) input.getValueByField("metadata");
```

Comment:
this wont work, because when it is a tick we do not have metadata information, and this bolt is triggering the search only when ticks. 
I will try to fix this asap